packages/haproxy/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.12.0"
+  changes:
+    - description: Update format version and enable secrets for sensitive fields. For more details, refer https://www.elastic.co/guide/en/fleet/current/agent-policy.html#agent-policy-secret-values
```

```suggestion
    - description: Enable secret for the `password` field and add dashboard-level filters.
```
We don't need to add the link here since it's already shown in Kibana, but I'm okay with keeping it if you prefer. We didn't do this in other secret enablement PRs.